composer.json

```
@@ -46,15 +46,15 @@
     },
     "require-dev": {
         "alcaeus/mongo-php-adapter": "^1.1",
-        "doctrine/cache": "^1.11 || ^2.0",
+        "doctrine/cache": "^1.11",
         "doctrine/dbal": "^2.13",
         "doctrine/doctrine-bundle": "^2.3",
         "doctrine/mongodb-odm": "^2.0",
         "doctrine/orm": "^2.9.6",
         "friendsofphp/php-cs-fixer": "^3.0",
-        "phpunit/phpunit": "^8.5",
-        "symfony/cache": "^4.4 || ^5.0",
-        "symfony/yaml": "^4.1"
+        "phpunit/phpunit": "^8.5 || ^9.5",
+        "symfony/cache": "^4.4 || ^5.3",
+        "symfony/yaml": "^4.1 || ^5.3"
```

```suggestion
        "symfony/yaml": "^4.4 || ^5.3"
```

Might as well make the Symfony component versions consistent, plus if bumping `symfony/cache` to not include EOM versions, `symfony/yaml` should have the same treatment applied.